lib/auth/usertasks/usertasksv1/service.go

```diff
@@ -182,11 +206,20 @@ func (s *Service) UpdateUserTask(ctx context.Context, req *usertasksv1.UpdateUse
 		return nil, trace.Wrap(err)
 	}
 
+	existingUserTask, err := s.backend.GetUserTask(ctx, req.GetUserTask().GetMetadata().GetName())
```

Comment:
Is the task guaranteed to exist here? I.e. this is actually "update" and not "upsert"?

Reply:
This one should only have `update` semantics.
So, if it doesn't exist it will return `not found` whether that check happens at the `backend.GetUserTask` or at the `backend.UpdateUserTask`.
We have another method in the `Service` that does the extra check.